src/datahike/index/hitchhiker_tree.cljc

```diff
@@ -119,14 +119,21 @@
 (defn -insert [tree ^Datom datom index-type op-count]
   (hmsg/insert tree (datom->node datom index-type) nil op-count))
 
+(defn- index-type->indices [index-type]
+  (case index-type
+    :eavt [0 1]
+    :aevt [0 1]
+    :avet [0 2]
```

Comment:
Is this true if we operate on the `avet` index itself? This is only true if we operate on `aevt`, right?

Reply:
@whilo Sorry, I don't understand the question.

Reply:
I meant the only case with `[0 2]`, `:avet`. I guess we do not work on the actual `avet` index in this case, otherwise it would be `[0 1]`.

Reply:
Yes, we don’t work on avet.

Reply:
Ok, can you explain why not? I would have guessed that we need to upsert in there as well(?). Sorry if this is a question with an obvious answer.

Reply:
Sorry my previous answer is based on a misunderstanding of your question. 
We *do* upsert on :avet but we need to use :a and :e, i.e., position 0 and 2 (and not 0 and 1) to correctly perform upserts on :avet.

Reply:
Thanks, that makes sense.